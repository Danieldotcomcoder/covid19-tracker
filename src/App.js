import { BrowserRouter as Router } from 'react-router-dom';

function App() {
  return (
    <div className="App">
      <Router>
        <h1> Hello </h1>
      </Router>
    </div>
  );
}

export default App;
