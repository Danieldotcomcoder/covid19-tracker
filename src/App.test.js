import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from './Redux/configureStore';
import App from './App';

jest.mock('./Redux/countries/FetchAPI.js');

describe('Main Page', () => {
  it('Renders App component', () => {
    const Appcomponent = renderer.create(
      <Provider store={store}>
        <App />
        ,
      </Provider>,
    ).toJSON();
    expect(Appcomponent).toMatchSnapshot();
  });
});
