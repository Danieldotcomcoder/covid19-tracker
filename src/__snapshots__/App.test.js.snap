// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Page Renders App component 1`] = `
Array [
  <div />,
  <section>
    <header
      className="home-main"
    >
      <img
        alt="arrowimage"
        src="back-button.png"
        title="aimg"
      />
      <h4
        className="2021"
      >
           2021
      </h4>
      <h5
        className="main-title"
      >
        Most Views
      </h5>
      <img
        alt="micropone"
        className="mic-image"
        src="microphone.png"
      />
      <div
        className="setimg"
      >
        <img
          alt="settings"
          src="settings.png"
        />
      </div>
    </header>
    <div
      className="home-middle"
    >
      <div
        className="Home-left"
      >
        <img
          alt="Banner view"
          className="App-map"
          src="europe.png"
        />
      </div>
      <div
        className="Home-right"
      >
        <h1
          className="cont-name"
        >
          Europe
        </h1>
        <p
          className="cont-info"
        >
          0
            Cases
        </p>
      </div>
    </div>
    <section
      className="country-info"
    >
      <div
        className="middle"
      >
        <h5
          className="statsbycountry"
        >
          STATS BY COUNTRY
        </h5>
        <input
          className="search"
          onChange={[Function]}
          placeholder="Search by Country"
          type="search"
        />
      </div>
      <ul
        className="grid"
      />
    </section>
  </section>,
  ",",
]
`;
